src/main/scala/gestalt/core/Trees.scala

```diff
@@ -131,11 +131,13 @@ trait Trees extends Params with TypeParams with
 
   def NewInstance: NewInstanceImpl
   trait NewInstanceImpl {
-    def apply(qual: Option[Tree], name: String, targs: List[TypeTree], argss: List[List[TermTree]]): TermTree
-    def apply(tpe: TypeTree, argss: List[List[TermTree]]): TermTree = {
-      val Trees.this.PathType(qual, name, targs) = tpe
-      apply(qual, name, targs, argss)
-    }
+    def apply(qual: Option[Tree],
+              name: String,
+              targs: List[TypeTree],
+              argss: List[List[TermTree]])(implicit unsafe: Unsafe): TermTree
```

Comment:
By the same reasoning, we should also change `InitCall` to adopt the same design.